LFS171x/fabric-material/basic-network/README.md

```diff
@@ -0,0 +1,14 @@
+## Basic Network Config
+
+Note that this basic configuration uses pre-generated certificates and
+key material, and also has predefined transactions to initialize a 
+channel named "mychannel".
+
+To regenerate this material, simply run ``generate.sh``.
+
+To start the network, run ``start.sh``.
+To stop it, run ``stop.sh``
+To completely remove all incriminating evidence of the network
+on your system, run ``teardown.sh``.
+
+<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>
```

Comment:
The source code should be licensed under APACHE-2.0 and the documentation under CC Attribution 4.0 International license.

Reply:
done